src/Symfony/Component/HttpFoundation/File/Exception/CantWriteFileException.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\HttpFoundation\File\Exception;
+
+/**
+ * Thrown when an error UPLOAD_ERR_CANT_WRITE occurred with UploadedFile.
```

Comment:
I would swap the word "error" and the name of the error.

Reply:
Updated